package.json

```diff
@@ -38,14 +31,17 @@
     "ethereumjs-testrpc": "^4.1.3",
     "ethereumjs-util": "^5.1.1",
     "mocha-lcov-reporter": "^1.3.0",
-    "solidity-coverage": "^0.2.3",
+    "solidity-coverage": "^0.2.4",
     "solidity-inspector": "^0.2.3",
     "solidity-sha3": "^0.4.1",
     "solium": "^0.5.5",
-    "truffle": "^3.4.9",
+    "truffle": "^3.4.11",
     "truffle-config": "^1.0.2",
     "truffle-expect": "0.0.3",
     "truffle-hdwallet-provider": "0.0.3",
-    "zeppelin-solidity": "^1.2.0"
+    "zeppelin-solidity": "^1.3.0"
+  },
+  "dependencies": {
+    "os-tmpdir": "^1.0.2"
```

Comment:
Why was this added?

Reply:
Oh! That got there when I was testing npm problem. I wasn't supposed to be saved. I'll remove

Reply:
Also there's a few deps that we do require that you've removed by accident, e.g. `solium` 😊